airflow/providers/amazon/aws/hooks/secrets_manager.py

```diff
@@ -0,0 +1,72 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+import base64
+import json
+from typing import Union, Optional
+from airflow.providers.amazon.aws.hooks.base_aws import AwsBaseHook
+
+
+class SecretsManagerHook(AwsBaseHook):
+    """
+    Interact with Amazon SecretsManager Service.
+
+    Additional arguments (such as ``aws_conn_id``) may be specified and
+    are passed down to the underlying AwsBaseHook.
+
+    .. see also::
+        :class:`~airflow.providers.amazon.aws.hooks.base_aws.AwsBaseHook`
+    """
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(client_type='secretsmanager', *args, **kwargs)
+
+    def get_secrets(self, secret_name: str) -> Union[str, bytes]:
```

Comment:
```suggestion
    def get_secret(self, secret_name: str) -> str:
```
This is only returning one secret, correct? And it will be decoded to string if it is a `SecretBinary`. So it is always of type `str`.

Reply:
^ Please also update the docs according that.

I think the title is not correct? Same in `get_secrets_as_dict`.

Reply:
base64.b64decode returns `bytes`, not `str` - we already have such type of discussion here and and I had to change it to `bytes`, docs about it - https://docs.python.org/3/library/base64.html.

Reply:
You are right, thanks - got it. :)